static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -105,12 +104,12 @@ function getDataSetQuery(widgetBuilderNewDesign: boolean): Record<DataSet, Widge
     },
     [DataSet.RELEASE]: {
       name: '',
-      fields: [`sum(${SessionMetric.SESSION})`],
+      fields: [`sum(session)`],
       columns: [],
       fieldAliases: [],
-      aggregates: [`sum(${SessionMetric.SESSION})`],
+      aggregates: [`sum(session)`],
       conditions: '',
-      orderby: widgetBuilderNewDesign ? `-sum(${SessionMetric.SESSION})` : '',
+      orderby: widgetBuilderNewDesign ? `-sum(session)` : '',
```

Comment:
Nice potential cleanup could be to use enum for session fields.
https://github.com/getsentry/sentry/blob/496e21347d491f2d06bc0b6eac5a08c9b2ec00a5/static/app/types/organization.tsx#L225-L229
Pitty it's registered together with an aggregate function, it could be reused even in more places if it wasn't.
Not a blocker.